GNUmakefile

```diff
@@ -50,16 +50,20 @@ cov:
 	open /tmp/coverage.html
 
 test: dev
-	go test -tags "$(GOTAGS)" -i ./...
-	go test -tags "$(GOTAGS)" -run '^$$' ./... > /dev/null
```

Comment:
tests run in 40 sec top to bottom including building. Adding a 2-3 sec latency for something that doesn't happen often and that you'll catch anyway is a high price IMO.